website/guides/terraform-provider-development-program.html.md

```diff
@@ -7,232 +7,137 @@ description: This guide is intended for vendors who're interested in having thei
 
-## What is a Terraform Provider?
+![Verified Provider Card](/assets/images/docs/verified-card.png)
 
-Terraform is used to create, manage, and manipulate infrastructure resources.
-Examples of resources include physical machines, VMs, network switches, containers, etc.
-Almost any infrastructure noun can be represented as a resource in Terraform.
+-> **Building your own provider?** If you're building your own provider and aren't interested in having HashiCorp officially verify and regularly monitor your provider, please refer to the [Writing Custom Providers guide](https://www.terraform.io/docs/extend/writing-custom-providers.html) and the [Extending Terraform](https://www.terraform.io/docs/extend/index.html) section.
 
-A provider is responsible for understanding API interactions with the underlying
-infrastructure like a cloud (AWS, GCP, Azure), a PaaS service (Heroku), a SaaS
-service (DNSimple, CloudFlare), or on-prem resources (vSphere). It then exposes
-these as resources users can code to. Terraform presently supports more than
-70 providers, a number that has more than doubled in the past 12 months.
 
-All providers integrate into and operate with Terraform exactly the same way.
-The table below is intended to help users understand who develops, maintains
-and tests a particular provider.
+## What is a Terraform Provider?
 
-![Provider Engagement Table](/assets/images/docs/engage-table.png)
+Terraform is used to create, manage, and interact with infrastructure resources of any kind. Examples of resources include physical machines, VMs, network switches, containers, etc. Almost any infrastructure noun can be represented as a resource in Terraform.
+
+A Terraform Provider represents an integration that is responsible for understanding API interactions with the underlying infrastructure, such as a public cloud service (AWS, GCP, Azure), a PaaS service (Heroku), a SaaS service (DNSimple, CloudFlare), or on-prem resources (vSphere). The Provider then exposes these as resources that Terraform users can interface with, from within Terraform a configuration. Terraform presently supports more than 70 providers, a number that has more than doubled in the past 12 months.
+
+All providers integrate into and operate with Terraform exactly the same way. The table below is intended to help users understand who develops, and maintains a particular provider.
+
+<table border="0" style="border-collapse: collapse; width: 100%;">
+<tbody>
+<tr style="height: 21px;">
+<td style="width: 12.4839%; height: 21px;"><strong>Tier</strong></td>
+<td style="width: 55.7271%; height: 21px;"><strong>Description</strong></td>
+<td style="width: 31.7889%; height: 21px;"><strong>Namespace</strong></td>
+</tr>
+<tr style="height: 21px;">
+<td style="width: 12.4839%; height: 21px;"><img src="./images/official-tier.png" alt="" /></td>
```

Comment:
be sure to include the badge image assets in ./images/... you can source from here: https://github.com/hashicorp/terraform/tree/master/website/docs/registry/providers/images

Reply:
Updated the paths